frontend/app/abtests/BundleVariant.scala

```diff
@@ -49,8 +48,8 @@ object BundleTier {
 
 
 object Distribution {
```

Comment:
Instead of creating a new design for A2, we should use the hasAdFree variable to show whether the variant has that feature or not.